static/templates/user_info_popover_content.hbs

```diff
@@ -56,7 +56,7 @@
         {{#if can_set_away}}
         <li>
             <a tabindex="0" class="set_away_status">
-                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}Set yourself as unavailable{{/tr}}
+                <i class="fa fa-minus-circle" aria-hidden="true"></i> {{#tr this}}set yourself as unavailable{{/tr}}
```

Comment:
I think this change is in error -- or did you make this change to see the test fail?

If the latter, can you move it to a separate commit that's labeled as something to not merge?

Reply:
yeah I think this small change in this file is part of another commit. Its a mistake it is merged here.